scripts/rosdistro_freeze_source

```diff
@@ -75,11 +77,18 @@ def main():
         args.dist_names = sorted(index.distributions.keys())
 
     for dist_name in args.dist_names:
+        dist_url = index.distributions[dist_name]['distribution']
+        if type(dist_url) == list:
+            if len(dist_url) != 1:
+                print("This script only works for distributions with a single distribution.yaml.", file=sys.stderr)
```

Comment:
Please include the distro name which has been skipped in the error message:

    print("The distribution '%s' has been skipped since this script only works with a single distribution.yaml" % dist_name, file=sys.stderr)

Please use single quotes wherever possible.